Models/TerritoryStats.ts

```diff
@@ -0,0 +1,31 @@
+class TerritoryStats {
+    ObjectID: number = 0;
+    Name: string = '';
+    Recovered: number = 0;
+    Tests: number = 0;
+    LastUpdated: number = 0;
+    SourceURL: string = '';
+    ActiveCases: number = 0;
+    Hospitalized: number = 0;
+    ICU: number = 0;
+    ICUVentilator: number = 0;
+    Deaths: number = 0;
+    CaseTotal: number = 0;
+    Vaccinated: number = 0;
+    constructor (objectId: number, name: string, recovered: number, tests: number, lastUpdated: number, sourceURL: string, activeCases: number, hospitalized: number, iCU: number, iCUVentilator: number, deaths: number, caseTotal: number, vaccinated: number) {
```

Comment:
These properties should be public and not set in the constructor.

Reply:
done